.github/workflows/build_and_tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Clone
       uses: actions/checkout@v3
     - name: Speculos dependencies
-      run: sudo apt-get update && sudo apt-get install -y qemu-user-static
```

Comment:
@lpascal-ledger @fbeutin-ledger Do you known why we are not using Speculos docker image here?

Reply:
No idea

Reply:
Temporally resolved by a fix on Speculos. Should be fixed later too.